Documentation/UserTsconfig/Setup.rst

```diff
@@ -28,6 +28,30 @@ setup.default.[someProperty]
     existing users is by :guilabel:`Reset Backend User Preferences` in the
     :guilabel:`Admin tools > Maintenance` section of the install tool.
 
+    .. code-block:: typoscript
+       :caption: EXT:site_package/Configuration/admin.tsconfig
+
+       [backend.user.isAdmin]
```

Comment:
Can you please indent the code in the condition. Additionally, we use in the docs consistently `[END]` instead of `[global]`.

I also suggest to write "an administrator" instead of "an admin" in the comment, avoiding "short names"
Also "backend" instead of "BE"